### ISSUE TITLE
Stuck at 0%

### ISSUE THREAD

**Issue by Animal6311** [DIGITALCRIMINALS/OnlyFans#395]
It starts to download the scrape but gets stuck at a few KB. I have plenty of space so that shouldn't be the issue. I have tried different OF subscriptions but I get the same problem for all. I've reinstalled and I am up to date with everything but the problem is still there.  Anyone know what I can do?

**Comment by yermumz** [comment]
I had a fresh install of the latest version, didn't like how it scrapes through all models even after selecting which model, so I went back to an older version and I ran into the same issue among a couple of others. I managed to fix the stuck downloads by installing all the modules listed in the old requirements.text with "pip install -r requirements.txt" 

Note: I don't know if it's going to help with stuck downloads in the latest version since I don't have the patience to wait through the other 300 models that I don't want to scrape. But it's probably worth a try.

Long story short, download the old requirements.txt, place it in the folder and run the command "pip install -r requirements.txt"

**Comment by Animal6311** [comment]
> I had a fresh install of the latest version, didn't like how it scrapes through all models even after selecting which model, so I went back to an older version and I ran into the same issue among a couple of others. I managed to fix the stuck downloads by installing all the modules listed in the old requirements.text with "pip install -r requirements.txt". Apparently it has a few modules on the list that poetry did not install.
> 
> Note: I don't know if it's going to help with stuck downloads in the latest version since I don't have the patience to wait through the other 300 models that I don't want to scrape. But it's probably worth a try.
> 
> Long story short, download the old requirements.txt, place it in the folder and run the command "pip install -r requirements.txt"

Yes! The old requirements.txt did work on the new version and everything is good now :) Thanks so much for your help.
